File: year2020/month09/day0921/ms_0921.cs
using System;

namespace algorithm_61
{
    class Program
    {
        static void BubbleSort(int[] data) 
        {
            for(int i = 0; i < data.Length-1; i++) 
            {
                for(int j = 0; j < data.Length-1; j++) 
                {
                    if(data[j] < data[j+1]) 
                    {
                        int temp = data[j];
                        data[j] = data[j+1];
                        data[j+1] = temp;
                    }
                }
 
                for(int j = 0; j < data.Length; j++) 
                {
                    Console.Write(data[j]);

                    if(j != data.Length)
                        Console.Write(' ');
                }

                Console.WriteLine();
            }
        }

        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int[] data = new int[10];
            int i = 0;

            foreach(string value in input)
                data[i++] = Convert.ToInt32(value);

            BubbleSort(data);
        }
    }
}

File: year2020/month09/day0921/ms_0921.cpp
#include <iostream>

using namespace std;

void bubbleSort(int data[], int size);

int main() {
	int data[10] = { 0, };

	for (int i = 0; i < 10; i++)
		scanf_s("%d", &data[i]);
	
	bubbleSort(data, sizeof(data) / sizeof(data[0]));

	return 0;
}

void bubbleSort(int data[], int size) {
	for (int i = 0; i < size - 1; i++) {
		for (int j = 0; j < size - 1; j++) {
			if (data[j] < data[j + 1]) {
				int temp = data[j];
				data[j] = data[j + 1];
				data[j + 1] = temp;
			}
		}

		for (int j = 0; j < size; j++) {
			cout << data[j];

			if (j != size)
				cout << ' ';
		}

		cout << endl;
	}
}